src/iexamine.cpp

```diff
@@ -53,6 +53,7 @@
 #include "overmapbuffer.h"
 #include "pickup.h"
 #include "player.h"
+#include "recipe_dictionary.h"
```

Comment:
You sure you need this include?

Reply:
yeah, had errors without.

Reply:
I think you just need recipe.h not recipe dictionary

Reply:
oh hm. lemme try that.